parm/config/gfs/config.wave

```diff
@@ -30,7 +30,7 @@ case "${waveGRD}" in
     export waveGRDN='1 2 3'
     export waveGRDG='10 20 30'
     export USE_WAV_RMP='YES'
-    export waveMULTIGRID='.true.'
+    export waveMULTIGRID='.false.'
```

Comment:
This option will have to be completely removed as this is a multi-grid set-up.  This option cannot be simply set to false.

Reply:
Will work on that.